src/graphql/mirror.js

```diff
@@ -1947,6 +2033,7 @@ export function _buildSchemaInfo(schema: Schema.Schema): SchemaInfo {
 }
 
 type UpdateId = number;
+type NetworkLogid = number;
```

Comment:
Can we call this `NetworkLogId` for consistency with `UpdateId`,
`ObjectId`, `RepoId`, `TaskId`, etc. elsewhere in SourceCred? (It’s true
that this differs from SQLite’s `rowid`, but the in-project convention
is stronger than the cross-project convention, and also SQLite’s choice
is motivated by SQL’s case-insensitivity, which doesn’t apply to us.)

Reply:
updated